examples/yaml-to-anydata-with-projection/yaml_to_anydata_with_projection.md

```diff
@@ -0,0 +1,14 @@
+# YAML to anydata conversion with projection
+
+The `data.yaml` library provides multiple APIs to selectively convert elements and attributes from a YAML source, 
+in the form of a string, byte array, or byte block stream, into a Ballerina record. 
+Using projection, users can selectively add attributes to records and limit the sizes of arrays. 
+In this example, the `password` attribute is excluded when creating the `DatabaseConfig` record, 
+and only the first two elements from the source are used to create the `remotePorts` array.
+
+
```

Comment:
Extra new line or do we have two in others?

Reply:
extra one. Will remove it.